### PR TITLE
profile structure integrated and logic updated

### PR DESCRIPTION
**Pull request by mushood123**

### CODE REVIEW

File: src/utils/schema.js [hunk 1]
```diff
@@ -29,6 +29,23 @@ export const signUpSchema = Yup.object().shape({
...
+    .min(18, 'too young')
+    .max(100, 'too old')
+    .required('Required'),
+  experience: Yup.number()
+    .min(1, 'too less')
+    .max(100, 'too much')
+    .required('Required'),
+  phone: Yup.string()
+    .required('Required')
+    .matches(/^\d+$/, 'The field must contain only numbers'),
```
Comment (muhammadawais1590) [inline src/utils/schema.js:50]: lets create a Constant for error labels

File: src/screens/Profile/Home/home.jsx [hunk 2]
```diff
@@ -1,18 +1,72 @@
...
+      />
+
+      <TouchableOpacity
+        onPress={confirmPressed}
+        style={styles.buttonContainer}
+        disabled={
+          errors?.age || errors?.name || errors?.experience || errors?.phone
+        }>
+        <Text style={styles.buttonText}>Confirm</Text>
+      </TouchableOpacity>
```
Comment (muhammadawais1590) [inline src/screens/Profile/Home/home.jsx:69]: lets create a button component

File: src/utils/schema.js (hunk 1 above)
Comment (mushood123) [reply src/utils/schema.js:50]: Sure.

File: src/screens/Profile/Home/home.jsx (hunk 2 above)
Comment (mushood123) [reply src/screens/Profile/Home/home.jsx:69]: Sure

**Pull request merged**